README.md

```diff
@@ -4,130 +4,267 @@
 
-## Setup
+## Validation
 
-1. Make sure [node.js](https://nodejs.org/) and [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) are installed on your system; the latest Long-Term Support (LTS) version is recommended for both (note that node v16 is not currently supported). [OpenSSL 1.1.1](https://www.openssl.org/) is also needed to validate certificate chains which could be present in issuer JSON Web Keys (`x5c` value); if absent, chain validation is skipped.
+The tool currently verifies proper encoding of the:
+ - QR code image (single file or split in chunks)
+ - Numeric QR data (header, content)
+ - SMART Health Card file (schema)
+ - JWS (schema, deflate compression, format, size limits, signature, issuer key retrieval, x5c cert chain validation)
+ - JWS payload (schema)
+ - FHIR bundle (basic schema validation).
+ - Issuer JSON Key Set (schema, algorithm, EC Curve, ID, type, usage)
 
-2. Get the source, for example using git:
+Validation of the FHIR bundle is currently limited. The tool validates a subset of the full FHIR schema; the behavior scoped by using the `--profile` option, or can be changed by modifying the `src/prune-fhir-schema.ts` script. Extensive tests and conformance to the [Vaccination & Testing Implementation Guide](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/) can be performed by the [FHIR validator](https://wiki.hl7.org/Using_the_FHIR_Validator) tool.
 
-                git clone -b main https://github.com/smart-on-fhir/health-cards-validation-SDK.git
-                cd health-cards-validation-SDK
+## Prerequisites
 
-3. Build the npm package:
+1. [node.js](https://nodejs.org/) and [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) are installed on your system; the latest Long-Term Support (LTS) version is recommended for both (note that node v16 is not currently supported). 
+1. [OpenSSL 1.1.1](https://www.openssl.org/) is also needed to validate certificate chains which could be present in issuer JSON Web Keys (`x5c` value); if absent, chain validation is skipped.
 
-                npm install
-                npm run build
+## Install via Github
 
-3. Optionally, run the tests:
+### Default branch:
+```
+npm install github:smart-on-fhir/health-cards-validation-SDK
+```
 
-                npm test
+### Specific release:
+```
+npm install github:smart-on-fhir/health-cards-validation-SDK#v1.0.0-4
+```
 
-The tests download and validate, among other things, the spec [examples](https://smarthealth.cards/examples/). A breaking spec change might invalidate the downloaded examples, which can be refreshed using:
+## Examples
 
-                npm run fetch-examples -- --force
+### Validate a `.smart-health-card` file (`data.smart-health-card`)
+```
+smc-validator --path data.smart-health-card --type healthcard
```

Comment:
shc-validator (instead of smc-validator); here and elsewhere in the page.

Reply:
Good catch, I've updated it to `shc` everywhere.